<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMediaUserDataWidget</class>
 <widget class="QGroupBox" name="CMediaUserDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Data:</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="isFavorite">
     <property name="text">
      <string>Is Favorite?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="hasBeenPlayed">
     <property name="text">
      <string>Has been Played?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Last Played Date (UTC):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="lastPlayedDate">
       <property name="currentSection">
        <enum>QDateTimeEdit::MonthSection</enum>
       </property>
       <property name="displayFormat">
        <string>M/d/yyyy hh:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="setTimeToNowBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="EmbySync.qrc">
         <normaloff>:/resources/clock.png</normaloff>:/resources/clock.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Playback Position:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTimeEdit" name="playbackPosition">
     <property name="displayFormat">
      <string>hh:mm:ss.z</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Play Count:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="playCount">
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="process">
       <property name="text">
        <string>&amp;Process</string>
       </property>
       <property name="icon">
        <iconset resource="EmbySync.qrc">
         <normaloff>:/resources/upload.png</normaloff>:/resources/upload.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply">
       <property name="text">
        <string>Apply From Here</string>
       </property>
       <property name="icon">
        <iconset resource="EmbySync.qrc">
         <normaloff>:/resources/arrowright.png</normaloff>:/resources/arrowright.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>isFavorite</tabstop>
  <tabstop>hasBeenPlayed</tabstop>
  <tabstop>lastPlayedDate</tabstop>
  <tabstop>setTimeToNowBtn</tabstop>
  <tabstop>playbackPosition</tabstop>
 </tabstops>
 <resources>
  <include location="EmbySync.qrc"/>
 </resources>
 <connections/>
</ui>
